src/components/table.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
Called this file table, in case we want to regroup different table elements into one file. What do you think?

Reply:
Great idea. Could be useful later 👍